src/status_im/chat/commands/core.cljs

```diff
@@ -34,6 +34,22 @@
   [type]
   (keyword (str (protocol/id type) "-button")))
 
+(defn add-chat-contacts
+  "Enrich command-message by adding contact list of the current private or group chat"
+  [contacts {:keys [public? group-chat] :as command-message}]
+  (if public?
```

Comment:
I would probably use `cond` here and `assoc` instead of `merge` (`merge` I would use only when the set second argument is dynamic)

```
(cond 
     public? command-message
     group-chat (assoc command-message :contacts ....)
     :else          (assoc ....))
```
`cond->` could also be used I guess

Reply:
👍 thanks @cammellos for the feedbacks! I prefer `cond` because of the short circuit.